### DISCUSSION TITLE
I can't write on file `texfile.log'

### DISCUSSION THREAD

**Discussion by johnhdavies** [MiKTeX/miktex#718] [Q&A]
I have run MiKTeX without problems for some time but now it has stopped working because it cannot create the log file. The message is:
! I can't write on file `texfile.log'.
This occurs in projects that have worked without problem in the past. I created a new directory and project but the same happened. I have tried creating the log file before running MiKTeX but it is deleted and the same message appears. I have tried running MiKTeX from within WinEdt and directly from the command line with the same outcome. The only change that I am aware of, since MiKTeX last worked, is a Windows update:
2021-02 Cumulative Update for Windows 10 Version 20H2 for x64-based Systems (KB4601319)
and an update for .NET Framework at the same time (KB4601050)
I re-installed MiKTeX and have tried both personal and system-wide installations but the problem remains.
Any suggestions to solve this, please?

**Comment by edocevoli** [comment]
Another process has locked the file for writing. Perhaps an anti malware program?

**Reply by johnhdavies** [reply]
You are abolutely correct! Thank you very much for responding so quickly.
Windows advised me to check my security settings when I installed the recent upgrade. I turned on the "Ransomware protection" and now realise that caused the problem. I have turned it off and MiKTeX now works as it should.
Many thanks again.
